### PR TITLE
Add species parameter to workflow

### PR DESCRIPTION
**Pull request by vlagrassa**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+services:
+  nemascan:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    entrypoint: nemascan-nxf.sh
+    environment:
+      - VCF_VERSION=20220216
+      - WORK_DIR=gs://caendr-nf-pipelines/workdir/rbv218
```
Comment (r-vieira) [inline docker-compose.yml]: hum....we might want to remove these values here, replace them with placeholders, and add some info in the `README.md` about how to populate them.

File: main.nf [hunk 2]
```diff
@@ -20,7 +20,7 @@ params.help = null
 params.debug = null
 download_vcf = null
 params.finemap = true
-params.species = "c_elegans"
+// params.species = "c_elegans"
```
Comment (r-vieira) [inline main.nf]: if appropriate, add a comment here explaining where the value is being injected.

File: nemascan-nxf.sh [hunk 3]
```diff
@@ -9,6 +9,9 @@ DEFAULT_VCF_VERSION="20220216"
 DEFAULT_GOOGLE_PROJECT="andersen-lab"
 DEFAULT_GOOGLE_ZONE="us-central1-a"
 DEFAULT_GOOGLE_SERVICE_ACCOUNT_EMAIL="nscalc-201573431837@andersen-lab.iam.gserviceaccount.com"
```
Comment (r-vieira) [inline nemascan-nxf.sh:11]: this code is not from us, but if possible, it shouldn't be exposing service accounts email addresses....

**Review by r-vieira — CHANGES_REQUESTED** [review]
see if you can 'sanitize' the values in the dockercompose (remove rbv218, etc).

File: docker-compose.yml (hunk 1 above)
Comment (vlagrassa) [reply docker-compose.yml]: For now, I've just removed the docker compose file entirely: c3b4a8252bfd84a4d05528f7040557522d949771

File: main.nf (hunk 2 above)
Comment (vlagrassa) [reply main.nf]: I've removed this line entirely: 810113e878f039598ff7d966414916c65f86111a

**Review by r-vieira — APPROVED** [review]
looks good for now.

**Review by r-vieira — APPROVED** [review]
looks good for now.

**Pull request merged**